pal/minigame/taobao_minigame.ts

```diff
@@ -0,0 +1,221 @@
+/*
+ Copyright (c) 2022-2023 Xiamen Yaji Software Co., Ltd.
```

Comment:
I think should use `-` to connect words for file name as others.

Reply:
The file name depends on the setting of cc.config.json, which uses the lower case of the platform name as the file name.